partials/git.md

```diff
@@ -0,0 +1,81 @@
+# Git
+
+Jeśli nie masz jeszcze zainstalowanego i/lub skonfigurowanego gita użyj jednej
+z podanych poniżej instrukcji.
+
+## GitHub Desktop
+
+**Tylko dla Windows i MacOS**
+
+GitHub Desktop jest to klient graficzny do obsługi gita w połączeniu z GitHubem.
+
+1. Ściągnij aplikację z [https://desktop.github.com/](https://desktop.github.com/)
+2. Rozpakuj archiwum
+3. Uruchom aplikację
+4. Naciśnij `Sign into Github.com`
+5. Wpisz swoje dane z [github.com](https://github.com)
+
+## Wiersz poleceń
+
+### Instalacja
+
+#### Windows
+
+Zainstaluj gita używając tego instalatora: [https://git-scm.com/download/windows](https://git-scm.com/download/windows)
```

Comment:
> [https://git-scm.com/download/windows](https://git-scm.com/download/windows)
Zamień na Bare URL
<https://git-scm.com/download/windows>

cc: https://twitter.com/piecioshka/status/1008448526890741765

Reply:
Ja to pisałem wcześniej przy użyciu edytora do md i z tego co pamiętam nie obsługiwał Bare URL właśnie. Bo nie ma tego w "specyfikacji" markdown i nie wszystkie jego implementacje wspierają. Ale ok zaraz pozamieniam wszędzie.

Reply:
Najlepszy edytor do *.md to taki, który hightlightuje składnie. Wyglądem się nie martw, ważne, że GitHub (Pages) to supportuje 👍